Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,132 @@ Meaning of columns:
-|-----------------------|----------------------|----------------------------|-----------------------|-------|
-| `IfcCartesianPoint`   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
-| `IfcPointOnCurve`     | :x:                  | -                          | -                     | [#359](https://www.github.com/tumcms/Open-Infra-Platform/issues/359) |
+| Entity                                | Covered              | Converter                  | Unit test             | Issue |
+|---------------------------------------|----------------------|----------------------------|-----------------------|-------|
+| `IfcAdvancedBrep`                     | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedBrepWithVoids`            | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedFace`                     | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcAlignmentCurve`                   | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcAnnotationFillArea`               | :x:                  | -                          | -                     | -     |
+| `IfcAxis1Placement`                   | :x:                  | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcAxis2Placement2D`                 | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcAxis2Placement3D`                 | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcAxis2PlacementLinear`             | :x:                  | `PlacementConverter`       | -                     | -     |
+| `IfcBlock`                            | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | -     |
+| `IfcBlossCurve`                       | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcBooleanClippingResult`            | :heavy_check_mark:   | `SolidModelConverter`      | :x:                   | -     |
+| `IfcBooleanResult`                    | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | -     |
+| `IfcBoundedCurve`                     | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcBoundingBox`                      | :heavy_check_mark:   | `SolidModelConverter`      | :x:                   | -     |
+| `IfcBoxedHalfSpace`                   | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcBSplineCurveWithKnots`            | :heavy_check_mark:   | `SplineConverter.`         | :x:                   |[#404 ](https://github.com/tumcms/Open-Infra-Platform/issues/404)|
+| `IfcBSplineSurfaceWithKnots`          | :x:                  | `SplineConverter.`         | -                     | -     |
```

Comment:
How come IfcBSplineWithKnots is listed twice?

Reply:
See, the second one has another name IfcBSpline**Surface**WithKnots

Reply:
Right - sorry for that.